extension/android/CMakeLists.txt

```diff
@@ -155,6 +155,28 @@ if(EXECUTORCH_BUILD_LLAMA_JNI)
     ${EXECUTORCH_ROOT}/examples/models/llama2/runner
     ${CMAKE_CURRENT_BINARY_DIR}/../../examples/models/llama2/runner
   )
+
+  if(NEURON_BUFFER_ALLOCATOR_LIB)
```

Comment:
@kirklandsign I've added protect under this flag. Please modify if you would rather do it another way. thanks!

Reply:
Looks good to me!